p11-kit/rpc-client.c

```diff
@@ -755,10 +756,17 @@ rpc_C_Initialize (CK_X_FUNCTION_LIST *self,
 	assert (module->vtable->connect != NULL);
 	ret = (module->vtable->connect) (module->vtable, reserved);
 
+	if (ret == CKR_OK) {
```

Comment:
Shouldn't be client further initialized to send `P11_RPC_PROTOCOL_VERSION_MAXIMUM`?

Reply:
It's not possible to send data, if `connect` returns error.

Reply:
It's sent in the default implementation of the `authenticate` method (`rpc_transport_authenticate`):
https://github.com/p11-glue/p11-kit/pull/371/files#diff-71598b22704d395f7e1134bcb2149f62123d7617365db84f3146cc5842115fe0R644

Reply:
Ah, then I missed it, thanks for pointing it out.